### DISCUSSION TITLE
included in the 'nav' configuration, which is not found in the documentation files.

### DISCUSSION THREAD

**Discussion by mrbarua** [squidfunk/mkdocs-material#7205] [Q&A] [answered]
### Context

Included in the 'nav' configuration, which is not found in the documentation files.

WARNING -  A reference to 'openldap/configuration/installation.md' is included in the 'nav' configuration, which is not found in the documentation files.
![2024-05-21 17_10_20-W2022 - VMware Workstation](https://github.com/squidfunk/mkdocs-material/assets/24254733/d0df0edf-329c-420d-8dc4-6fb8198cdd16)

### Bug description

My Documentation  Folder Structure under docs: 

docs :
Openldap/configuration/openldap.md

#Page tree

- WAM:
  - OpenLDAP:
  - OpenLDAP CLI: opendalp/configuration/openldap.md

But I have the following Warning, Could you tell me why? although in the end-user-page I can access the page the Warning bothers me (Why warning)

$ mkdocs serve
INFO    -  Building documentation...
INFO    -  Cleaning site directory
INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - openldap\configuration\openldap.md

WARNING -  A reference to 'openldap/configuration/installation.md' is included in the 'nav' configuration, which is not found in the documentation files.
files.
INFO    -  Documentation built in 2.33 seconds
INFO    -  [17:10:10] Watching paths for changes: 'docs', 'mkdocs.yml'
INFO    -  [17:10:10] Serving on http://127.0.0.1:8000/

### Related links

- [Reporting a bug](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/)
-

### Reproduction

![2024-05-21 17_10_20-W2022 - VMware Workstation](https://github.com/squidfunk/mkdocs-material/assets/24254733/3a4cbbd9-f936-4e0b-a4df-6cd0615290cb)

### Steps to reproduce

NA

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) using the [built-in info plugin](https://squidfunk.github.io/mkdocs-material/plugins/info/).

**Comment by squidfunk** [comment]
Thanks for reporting. This is not a bug, but MkDocs behavior. Converting to discussion.

**Reply by mrbarua** [reply]
Does it mean I can ignore this warning? Does Material-mkdocs support double folder path nav? 
Maybe this warning was raised due to a double folder path? 
opendalp/configuration/openldap.md
openldap : a folder
configuration : a folder
openldap.md : a file

**Reply by alexvoss** [reply, marked as answer]
You have not shared with us how you set up your navigation. It might make sense  for you to have a look through the [MkDocs documentation on pages and navigation](https://www.mkdocs.org/user-guide/writing-your-docs/#configure-pages-and-navigation). There is no problem with having nested directory structures, it simply that there is a discrepancy between what is defined in the navigation and what Markdown files MkDocs found, so it warns about that. You can ignore it, though most likely it does point to a mistake you made somewhere.

**Reply by mrbarua** [reply]
Thank you so much @alexvoss 
Copy-paste sometimes shit.
When I manually typed all the Navigation pages, it did not give any more "Warning"
Thanks again.
